### PR TITLE
TECH-188: Create test configuration for "/{GovStackInstance}/{memberClass}/{memberCode}/{applicationCode}/allowedMethods" endpoint 

### PR DESCRIPTION
**Pull request by mgrochalskisoldevelo**
Created mocks and Cucumber tests for  "/{GovStackInstance}/{memberClass}/{memberCode}/{applicationCode}/allowedMethods" endpoint 

**Description**
Create test configuration for /{GovStackInstance}/{memberClass}/{memberCode}/{applicationCode}/allowedMethods endpoint from https://github.com/GovStackWorkingGroup/bb-information-mediator/blob/main/api/govstack_im_service_metadata_api-0.3-swagger.json.  This should include all the steps that are pointed out in the parent ticket.

TICKET: https://govstack-global.atlassian.net/browse/TECH-188

### CODE REVIEW

File: test/features/allowed_methods.feature [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Feature: This endpoint is used to list allowed REST services and endpoints for a service provider.	
+
+  List of allowed REST services and endpoints for a service provider
+
+  Request endpoint: GET /{GovStackInstance}/{memberClass}/{memberCode}/{applicationCode}/allowedMethods
+
+  Scenario: Successfully retrieved the list of allowed REST services and endpoints for a service provider
+    Given Wants to retrieve the list of allowed REST services and endpoints for a service provider
+    When The request to retrieve the list of allowed REST services and endpoints for a service provider is sent
+    Then The operation returns the list of allowed REST services and endpoints for a service provider
```
Comment (KarolinaKopacz) [inline test/features/allowed_methods.feature]: Please add empty line below

**Review by KarolinaKopacz — CHANGES_REQUESTED** [review]

File: test/features/allowed_methods.feature (hunk 1 above)
Comment (mgrochalskisoldevelo) [reply test/features/allowed_methods.feature]: Thanks, I added the line in second commit

**Review by KarolinaKopacz — APPROVED** [review]

File: examples/mock/mockoon-bb-information-mediator.json [hunk 2]
```diff
@@ -63,23 +63,52 @@
...
           "filePath": "",
           "databucketID": "",
           "sendFileAsBody": false,
-          "rules": [],
-          "rulesOperator": "OR",
+          "rules": [
+            {
+              "target": "params",
+              "modifier": "GovStackInstance",
+              "value": "^.{1,}$",
```
Comment (aleksander-reitsakas) [inline examples/mock/mockoon-bb-information-mediator.json]: Should be identifier

File: examples/mock/mockoon-bb-information-mediator.json [hunk 3]
```diff
@@ -63,23 +63,52 @@
...
+              "invert": false,
+              "operator": "regex"
+            },
+            {
+              "target": "params",
+              "modifier": "memberClass",
```
Comment (aleksander-reitsakas) [inline examples/mock/mockoon-bb-information-mediator.json:91]: Should be identifier

File: examples/mock/mockoon-bb-information-mediator.json [hunk 4]
```diff
@@ -63,23 +63,52 @@
...
+              "value": "^.{1,}$",
+              "invert": false,
+              "operator": "regex"
+            },
+            {
+              "target": "params",
+              "modifier": "memberCode",
+              "value": "^.{1,}$",
```
Comment (aleksander-reitsakas) [inline examples/mock/mockoon-bb-information-mediator.json]: Should be alphanumeric string

File: examples/mock/mockoon-bb-information-mediator.json [hunk 5]
```diff
@@ -63,23 +63,52 @@
...
+              "invert": false,
+              "operator": "regex"
+            },
+            {
+              "target": "params",
+              "modifier": "applicationCode",
+              "value": "^.{1,}$",
```
Comment (aleksander-reitsakas) [inline examples/mock/mockoon-bb-information-mediator.json]: Should be alphanumeric string

File: examples/mock/mockoon-bb-information-mediator.json (hunk 3 above)
Comment (aleksander-reitsakas) [reply examples/mock/mockoon-bb-information-mediator.json:91]: sry, next line, not this

**Review by KarolinaKopacz — APPROVED** [review]

**Review by aleksander-reitsakas — APPROVED** [review]

**Review by KarolinaKopacz — APPROVED** [review]

**Pull request merged**